tests/client_tests/go/basic_queries.go

```diff
@@ -0,0 +1,70 @@
+
+package main
+
+import (
+    "log"
+    "flag"
+    "fmt"
+    "github.com/jackc/pgx"
+)
+
+
+func main() {
+    hosts := flag.String("hosts", "", "CrateDB hostname")
+    port := flag.Int("port", 5432, "CrateDB postgres port")
+    flag.Parse()
+
+	pgxConfig := pgx.ConnConfig{
+		Host:     *hosts,
+		Port:     uint16(*port),
+		Database: "doc",
+		User:     "crate"}
+    conn, err := pgx.Connect(pgxConfig)
+    if err != nil {
+        log.Fatal(err)
+        return
+    }
+    var name string
+    err = conn.QueryRow("select name || $1 from sys.cluster", "foo").Scan(&name)
+    if err != nil {
+        log.Fatal(err)
+        return
+    }
+    fmt.Println(name)
+    commandTag, err := conn.Exec("create table t1 (x int)")
+    if err != nil {
+        log.Fatal(err)
```

Comment:
How are the results checked?

Reply:
If the statement causes an error it exists like this:

```
2017/12/01 13:33:54 cannot convert 2 to Text
exit status 1
Failure running: go run tests/client_tests/go/basic_queries.go --hosts 127.0.0.1 --port 5432
```

These tests are just simple "shouldn't run into an error" cases. The commandTag, results are tested in CrateDB itself.